src/anbox/platform/sdl/platform.cpp

```diff
@@ -370,6 +315,49 @@ void Platform::process_input_event(const SDL_Event &event) {
 #endif
 }
 
+bool Platform::calculate_touch_coordinates(const SDL_Event &event,
+                                                 std::int32_t &x,
```

Comment:
Indentation is incorrect here

Reply:
I have used window_moved() and window_resized() as models, but now I see that in other file the convention seem different, I'll fix this